cmd/localCI/vendor/github.com/BurntSushi/toml/cmd/toml-test-decoder/README.md

```diff
@@ -0,0 +1,13 @@
+# Implements the TOML test suite interface
+
+This is an implementation of the interface expected by
+[toml-test](https://github.com/BurntSushi/toml-test) for my
```

Comment:
Hi @iphutch  thanks for reviewing, the ```vendor``` directory contains third party code that we can't modify